server/package.json

```diff
@@ -9,8 +9,8 @@
 		"migration:run": "typeorm-ts-node-esm migration:run -d ./src/db.ts",
 		"start": "node dist/index.js",
 		"start:watch": "tsc -w & nodemon dist/index.js",
-		"test": "jest --runInBand",
-		"test:watch": "jest --watchAll --runInBand"
+		"test": "jest",
```

Comment:
why ? it will cause some problems no?

Reply:
Not really, --runInBand run all tests serially in the current process, it's good for depuration mode, but it doesn't work on the make exec, so will be useless.